src/views/menus/AccountMenu/AccountMenu.tsx

```
@@ -56,131 +56,127 @@ import { SubaccountActions } from './SubaccountActions';
-        hdKey,
-    } = useAccounts();
-    const { registerAffiliate } = useSubaccount();
-
-    let displayAddress: string | undefined;
-    if (walletInfo?.name === WalletType.Phantom) {
-        displayAddress = truncateAddress(address, '');
-    } else {
-        displayAddress = truncateAddress(address, '0x');
-    }
+  const stringGetter = useStringGetter();
+  const { mintscanBase } = useURLConfigs();
```

ESLint

[error] 'mintscanBase' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)